int16 thrusterLeftCommand
int16 thrusterRightCommand
